src/_guides/language/language-tour.md

```diff
@@ -1333,45 +1337,62 @@ For example:
 enableFlags(bold: true, hidden: false);
 ```
 
-Although it often makes sense to place positional arguments first,
-named arguments can be placed anywhere in the argument list
-when it suits your API:
+<a id="default-parameters"></a>
+To define a default value for a named parameter besides `null`,
+use `=` to specify a default value.
+The specified value must be a compile-time constant.
+For example:
 
-<?code-excerpt "misc/lib/language_tour/functions.dart (named-arguments-anywhere)"?>
+<?code-excerpt "misc/lib/language_tour/functions.dart (named-parameter-default-values)"?>
 ```dart
-repeat(times: 2, () {
-  ...
-});
-```
+/// Sets the [bold] and [hidden] flags ...
+void enableFlags({bool bold = false, bool hidden = false}) {...}
 
-{{site.alert.tip}}
-  If a parameter is optional but can't be `null`,
-  provide a [default value](#default-parameter-values).
-{{site.alert.end}}
+// bold will be true; hidden will be false.
+enableFlags(bold: true);
+```
 
-Although named parameters are a kind of optional parameter,
-you can annotate them with `required` to indicate
-that the parameter is mandatory—that users
-must provide a value for the parameter.
-For example:
+If you instead want a named parameter to be mandatory,
+requiring callers to provide a value for the parameter,
+annotate them with `required`:
 
 <?code-excerpt "misc/lib/language_tour/functions.dart (required-named-parameters)" replace="/required/[!$&!]/g"?>
-{% prettify dart tag=pre+code %}
+```dart
 const Scrollbar({super.key, [!required!] Widget child});
-{% endprettify %}
+```
 
 If someone tries to create a `Scrollbar`
 without specifying the `child` argument,
 then the analyzer reports an issue.
 
-{% comment %}
-NULLSAFE: Rewrite this section.
-{% endcomment %}
+{{site.alert.note}}
+  A parameter marked as `required`
+  can still be nullable:
```

Comment:
**question:** Is it best to say `is nullable` everywhere? Would it be more clear to say `...can be set to null`?

Reply:
This is talking more about the type of the variable, not the variable itself, so it seems better to stick to nullable.